src/OrchardCore.Themes/TheTheme/Views/Shared/LoginMenu.cshtml

```diff
@@ -1,3 +1,9 @@
+@using Microsoft.AspNetCore.Authentication
+@inject IAuthenticationSchemeProvider SchemeProvider
+
+@{ 
+    var providers = await SchemeProvider.GetRequestHandlerSchemesAsync();
```

Comment:
`GetExternalAuthenticationSchemesAsync`

Reply:
That will work if we use `ISignInManager`

Reply:
Also I don't want to depend on `OrchardCore.Users`

Reply:
Then just remove this link. We'll provide it correctly when we have time.

Reply:
Till now I'm not sure what's the wrong with using `GetRequestHandlerSchemesAsync()`?!!

Reply:
Till now I'm not sure what's the wrong with using `GetRequestHandlerSchemesAsync()`?!!

Reply:
It's just that it's testing something different than the one that the page it links to is showing. Maybe the lists are both empty by chance, and when you will actually have some other schemes handled we'll display the link, and it will still be empty on this page.

Because this feature is very rare, and because right now the link is only available here, and it doesn't make sense here, I suggest to remove it.

Reply:
Looking for a better way to support this or I will remove as you suggested above

Reply:
BTW removing the link is not solving the issue while the page is still accessible by URL

Reply:
BTW removing the link is not solving the issue while the page is still accessible by URL